docs/doc/30-reference/20-functions/60-conversion-functions/tofloat32.md

```diff
@@ -0,0 +1,22 @@
+---
+title: toFloat32
```

Comment:
Hello, I think we should just keep `cast.md` doc and remove all `toxxx.md`

1. Use `title_includes` to cover all other aliases, eg: https://github.com/datafuselabs/databend/blob/f799b6e6114d00cb48d49f9c611105c052bf78de/docs/doc/30-reference/20-functions/30-datetime-functions/subtractinterval.md

2. Create a link to indicate the datatypes that we supported in `cast.md`:  https://github.com/datafuselabs/databend/blob/8dcb757f2da1658f12e4854cf0b14dee7099e8f8/docs/doc/30-reference/10-data-types/index.md

 

```

```